### ISSUE TITLE
Module Manage tags 

### ISSUE THREAD

**Issue by christian2392** [ElevenPaths/thethe#26]
Error message: You dont have permissions to manage tag. 

Where do i manage the permissions if only allow to create a user and we dont definitions roles.

**Comment by deibit** [comment]
Can you provide context on the issue? A screenshot would help.

**Comment by christian2392** [comment]
Dear ElevenPaths

Good morning, im trying to create a new tag but show me the error message “You dont have permissions to manage tags” when put the button “create”

***@***.***D71EE9.4FC2AAD0]


Aditiional, please help me with a dubts how do i get the api keys for diario dinoflux and tacyt? . and the finally is it possible to increase the level of processing when running a plugin, it takes a long time to load?

Thanks so much


Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: David ***@***.***>
Enviado: lunes, 22 de marzo de 2021 6:51
Para: ***@***.***>
CC: ***@***.***>; ***@***.***>
Asunto: Re: [ElevenPaths/thethe] Module Manage tags (#26)


Can you provide context on the issue? A screenshot would help.

**Comment by deibit** [comment]
1) All odds, there is an issue with permission on initial user setup, which does not apply the right set. Will see it, but do not expect a short response.

2) For DIARIO keys, you have all the info available in the project page: https://diario.elevenpaths.com/

Cannot help with dinoflux and tacyt.

3) I do not understand what you mean with "level of processing". Please, elaborate it.

**Comment by christian2392** [comment]
Ok, I Will wait your answer for this issue.

3. I was referring to, if it is possible to increase the level of processing when running a plugin, the response time is much higher than that evidenced in the demonstration videos. My computer has 32gb ram 2tb disk and 8 processors. How to improve the processing speed of plugins?


2. For DIARIO keys, you have all the info available in the project page: https://diario.elevenpaths.com/    “I already tried to register but they still haven't activated my account, please kindly help me to activate the account ***@***.***"”


Thanks so much


Great regards
Christian Sevilla M.


Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: David ***@***.***>
Enviado: lunes, 22 de marzo de 2021 9:46
Para: ***@***.***>
CC: ***@***.***>; ***@***.***>
Asunto: Re: [ElevenPaths/thethe] Module Manage tags (#26)


  1.  All odds, there is an issue with permission on initial user setup, which does not apply the right set. Will see it, but do not expect a short response.
  2.  For DIARIO keys, you have all the info available in the project page: https://diario.elevenpaths.com/

Cannot help with dinoflux and tacyt.

  1.  I do not understand what you mean with "level of processing". Please, elaborate it.

**Comment by deibit** [comment]
Hi Christian. 

Account activation on DIARIO is done by hand, one by one. 

Regarding speeding plugin runtime, most of the tasks are not bound by CPU but network responses. Every request to a service has to be processed by a third party entity. Hence, you don't need a powerful set to run thethe, all is done by network requests to other services (even if the service is a command line tool in the container, think 'whois').

**Comment by christian2392** [comment]
Ok, thanks for the clarification. I have installed a server MISP. Is it posible synchonyze misp with thethe as with thehive?


Thanks

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10


De: David García ***@***.***>
Enviado: Monday, March 22, 2021 11:52:15 AM
Para: ElevenPaths/thethe ***@***.***>
Cc: christian2392 ***@***.***>; Author ***@***.***>
Asunto: Re: [ElevenPaths/thethe] Module Manage tags (#26)


Hi Christian.

Account activation on DIARIO is done by hand, one by one.

Regarding speeding plugin runtime, most of the tasks are not bound by CPU but network responses. Every request to a service has to be processed by a third party entity. Hence, you don't need a powerful set to run thethe, all is done by network requests to other services (even if the service is a command line tool in the container, think 'whois').

**Comment by deibit** [comment]
> Ok, thanks for the clarification. I have installed a server MISP. Is it posible synchonyze misp with thethe as with thehive? 

Currently, there is no support to directly export/import from any platform, sorry.

**Comment by christian2392** [comment]
Is there any plugin for validate if a IoC is in a blacklist? For example MXtoolbox?




Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: David ***@***.***>
Enviado: martes, 23 de marzo de 2021 11:24
Para: ***@***.***>
CC: ***@***.***>; ***@***.***>
Asunto: Re: [ElevenPaths/thethe] Module Manage tags (#26)


Ok, thanks for the clarification. I have installed a server MISP. Is it posible synchonyze misp with thethe as with thehive?

Currently, there is no support to directly export/import from any platform, sorry.

**Comment by deibit** [comment]
IIRC, there are some of them that could bring you info on blocklists. In any case, writing a thethe plugin is not as hard as it may see. Wanna give a try for mxtoolbox?

**Comment by christian2392** [comment]
Yes, it is one of the main engines and additionally I have a subscription so I would take advantage of the API. Among many analyzed Securitytrails is also a good api for the integration and obtaining of subdomains, if possible your help would be appreciated.

As additional points for improvement, I would like to suggest the following that I believe will add much more value to the tool:
1. The results of the search bar, it would be good to generate a link to go directly to the project where it is located
2. A button that allows the export of all the indicators of commitment found in the project
3. Creation and management of multiple users by roles.
4. Module for generating project report (pdf)
5. An option to be able to execute or mark all the plugins to one or more indicators at the same time

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: David ***@***.***>
Enviado: jueves, 25 de marzo de 2021 1:34
Para: ***@***.***>
CC: ***@***.***>; ***@***.***>
Asunto: Re: [ElevenPaths/thethe] Module Manage tags (#26)


IIRC, there are some of them that could bring you info on blocklists. In any case, writing a thethe plugin is not as hard as it may see. Wanna give a try for mxtoolbox?

**Comment by deibit** [comment]
Hi Christian, thanks for your feedback. We really appreciate that. Almost all the features you are proposing are being implemented/addressed in thethe v2 which hasn't been released as open source yet (and no date is scheduled yet).
